staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/webhook_converter.go

```diff
@@ -0,0 +1,140 @@
+package conversion
+
+import (
+	"errors"
+	"fmt"
+
+	"context"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/apiserver/pkg/admission/plugin/webhook/config"
```

Comment:
this deserves a more generic location. But good that it is reused :+1:

Reply:
Sorry you looked at the old code. I just pushed a new one that move all of these to a new location, please take another look.